Tasks/powerShell/task.json

```diff
@@ -50,13 +64,25 @@
             "required":false,
             "helpMarkDown": "Current working directory when script is run.  Defaults to the folder where the script is located.",
             "groupName":"advanced"
+        },
+        {
+            "name": "script",
+            "type": "multiLine",
+            "label": "Script",
+            "defaultValue": "Write-Host 'Hello World'",
+            "required": true,
+            "helpMarkDown": "Write your scripts and have fun",
+            "visibleRule": "type = InlineScript",
+            "properties": {
+                "rows": 5
+            }
         }
     ],
     "instanceNameFormat": "Powershell: $(scriptName)",
```

Comment:
In case of inline script, it will be empty. I will confirm it once with the spec what has to be shown in case inline script.